Updating 3D map view advanced configuration (#8223)
qgis/QGIS-Documentation: docs/user_manual/map_views/3d_map_view.rst

* |unchecked| :guilabel:`Show labels`: Toggles map labels on/off
* |unchecked| :guilabel:`Show map tile info`: Include border and tile
  numbers for the terrain tiles (useful for troubleshooting terrain issues)
* |unchecked| :guilabel:`Show bounding boxes`: Show 3D bounding boxes
  of the terrain tiles (useful for troubleshooting terrain issues)
* |unchecked| :guilabel:`Show camera's view center`
* |unchecked| :guilabel:`Show camera's rotation center`
* |unchecked| :guilabel:`Show light sources`: shows a sphere at light source origins,
  allowing easier repositioning and placement of light sources relative to the scene contents
* |unchecked| :guilabel:`Show frames per second (FPS)`
* |unchecked| :guilabel:`Show debug overlay`: visual overlay which displays
  some useful debugging and profiling information.
  This allows in particular to quickly see the frame graph and the scene graph.

.. _eye_dome_lighting:

* |unchecked| :guilabel:`Show Eye Dome Lighting` (EDL):
  a post processing effect which compares each pixel's depth (distance off the camera)
  with neighbor pixels depth and highlights the edges according to the depth difference.

>   Affects the whole scene. Following parameters can be controlled:

Reviewer: EDL highlights depth perception better than SSAO, so imho this could be reflected on the descriptions.

Author: I thought it was the other way. Can you provide a suggestion, please? Thanks

Author: Thanks @uclaros

> * |unchecked| :guilabel:`Show Eye Dome Lighting` (EDL):
>   a post processing effect which compares each pixel's depth (distance off the camera)
>   with neighbor pixels depth and highlights the edges according to the depth difference.
>   Affects the whole scene. Following parameters can be controlled:

Author: ```suggestion
* |unchecked| :guilabel:`Show Eye Dome Lighting` (EDL):
  a post processing effect which enhances depth perception.
  Each pixel's depth (distance off the camera) is compared to its neighboring pixels' depth
  and gets highlighted according to that depth difference, making the edges stand out.
  Affects the whole scene and can be combined with :ref:`Screen Space Ambient Occlusion <ambient_occlusion>`.
  Following parameters can be controlled:
```

  * :guilabel:`Lighting strength`: increases the contrast, allowing for better depth perception

>     (needs to be adjusted when zooming in)

Reviewer: >  (needs to be adjusted when zooming in)

why is that? I've never felt the need to adjust!

Author: _note: I'm not a point cloud user._
Because devs do not necessarily update their description before merging, I try to grab info in description, discussion, commits but this info comes from the [top message](https://github.com/qgis/QGIS/pull/39760#issue-734726434) (also landed in [changelog](https://qgis.org/en/site/forusers/visualchangelog318/index.html#feature-eye-dome-lighting)) and I couldn't find any objection. Maybe we improved it meanwhile?
```suggestion
```

Reviewer: :+1:

  * :guilabel:`Lighting distance`: represents the distance of the used pixels off the center pixel
    and has the effect of making edges thicker.

.. _ambient_occlusion:

* Add screen-space |unchecked| :guilabel:`Ambient Occlusion` (SSAO):

>   a post processing effect which nicely renders depth perception of the data in 3D
>   and can be combined with Eye dome Lighting.

Author: ```suggestion
  a post processing effect which also enhances depth perception
  by applying a darker shading to areas which are less exposed to ambient lighting.
  Affects the whole scene and can be combined with :ref:`Eye dome Lighting <eye_dome_lighting>`.
```

  Following parameters can be controlled:

  * :guilabel:`Radius`: how far we will reach to calculate ambient occlusion
  * :guilabel:`Intensity`: how strong the effect should be (higher values make things darker)
  * :guilabel:`Occlusion threshold`: how many neighboring points need to be occluded for the effect to appear
    (lower values than 50% will make the output darker, but possibly providing greater range of occlusion)

.. _figure_3dmaps_edl_ssao:

.. figure:: img/3dmap_edl_ssao.png
   :align: center

   Rendering Point clouds in 3D map using Eye Dome Lighting (EDL) and/or Screen-Space Ambient Occlusion (SSAO)

   From top, left to right: No effect -- SSAO only -- EDL only -- SSAO and EDL

* |unchecked| :guilabel:`Debug Shadow Map`: renders the scene as a red-black image
  from the point of view of the light used for shadows (for troubleshooting). 
  The widget is set with a proportional :guilabel:`Size` to the 3D map view's,
  and docked in a :guilabel:`Corner`.

> * |unchecked| :guilabel:`Debug Depth Map`: renders the scene as a red-black image
>   with nearer pixels being darker (for troubleshooting).

Reviewer: It's not red-black, it's scaled from black for the near plane to white for the far plane
```suggestion
* |unchecked| :guilabel:`Debug Depth Map`: renders the scene's depth map as an  image
  with nearer pixels being darker (for troubleshooting).
```

Author: Thanks for correction. Also picked at https://github.com/qgis/QGIS/pull/39760#issuecomment-722957387
